content/en/releases/patch-releases.md

```diff
@@ -78,13 +78,23 @@ releases may also occur in between these.
 
 | Monthly Patch Release | Cherry Pick Deadline | Target date |
 | --------------------- | -------------------- | ----------- |
-| April 2022            | 2022-04-08           | 2022-04-13  |
-| May 2022              | 2022-05-13           | 2022-05-18  |
+| May 2022              | 2022-05-20           | 2022-05-24  |
```

Comment:
move one week ahead due to kubecon

so the CP deadline will be on Friday (the last day of Kubecon) and the cut in the following week